interactors/src/main/java/com/thefuntasty/interactors/disposables/SingleDisposablesOwner.kt

```diff
@@ -32,12 +32,19 @@ interface SingleDisposablesOwner {
      * can be disabled by passing false to [SinglerConfig.disposePrevious]
      * method.
      *
-     * @param args Arguments used for initial interactor initialisation.
      * @return disposable of internal [Single]. This disposable is disposed
      * automatically. It might be used to dispose interactor when you need
      * to dispose it in advance on your own.
      */
-    fun <ARGS, T> BaseSingler<ARGS, T>.execute(args: ARGS): Disposable = execute(args, { })
+    // variant A of execute without params
+    fun <T> BaseSingler<Unit, T>.execute(result: SinglerConfig.Builder<T>.() -> Unit): Disposable {
+        return execute(Unit,result)
+    }
+
+    // variant B of execute without params
+    fun <T> BaseSingler<Unit, T>.executeNoArgs(result: SinglerConfig.Builder<T>.() -> Unit): Disposable {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/217c12b6260e5ed0fabdc621142369ea441e753d/interactors/src/main/java/com/thefuntasty/interactors/disposables/SingleDisposablesOwner.kt#L45"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>